XIVSlothCombo/Combos/PvE/DRG.cs

```diff
@@ -232,7 +232,7 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                         return OriginalHook(ChaosThrust);
                     }
 
-                    if (lastComboMove == OriginalHook(ChaosThrust) && LevelChecked(WheelingThrust))
```

Comment:
The originalhook makes it so it works for the lower levels aswell.
Changing this makes it so it only works once u got the upgraded skill